pkg/syncclient/sync_client.go

```diff
@@ -209,6 +209,7 @@ func (s *SyncerClient) connect(cxt context.Context) error {
 			return err
 		}
 		tlsConfig := tls.Config{Certificates: []tls.Certificate{cert}}
+		tlsConfig.MaxVersion = tls.VersionTLS12
```

Comment:
What's this doing here?  Needs a comment to explain it if nothing else

Reply:
go 1.13 defaults to TLS13, which causes some UT failures

Reply:
Please create a Core story to revisit this and allow/enable TLS 1.3.